py/server/deephaven/table.py

```diff
@@ -2156,19 +2156,19 @@ def update_by(self, ops: Union[UpdateByOperation, List[UpdateByOperation]],
     def slice(self, start: int, stop: int) -> Table:
         """Extracts a subset of a table by row positions into a new Table.
 
-          If both the start and the stop are positive, then both are counted from the beginning of the table.
-          The start is inclusive, and the stop is exclusive. slice(0, N) is equivalent to :meth:`~Table.head` (N)
-          The start must be less than or equal to the stop.
+            If both the start and the stop are positive, then both are counted from the beginning of the table.
+            The start is inclusive, and the stop is exclusive. slice(0, N) is equivalent to :meth:`~Table.head` (N)
+            The start must be less than or equal to the stop.
 
-          If the start is positive and the stop is negative, then the start is counted from the beginning of the
-          table, inclusively. The stop is counted from the end of the table. For example, slice(1, -1) includes all
-          rows but the first and last. If the stop is before the start, the result is an empty table.
+            If the start is positive and the stop is negative, then the start is counted from the beginning of the
+            table, inclusively. The stop is counted from the end of the table. For example, slice(1, -1) includes all
+            rows but the first and last. If the stop is before the start, the result is an empty table.
 
-          If the start is negative, and the stop is zero, then the start is counted from the end of the table,
-          and the end of the slice is the size of the table. slice(-N, 0) is equivalent to :meth:`~Table.tail` (N).
+            If the start is negative, and the stop is zero, then the start is counted from the end of the table,
+            and the end of the slice is the size of the table. slice(-N, 0) is equivalent to :meth:`~Table.tail` (N).
 
-          If the start is negative and the stop is negative, they are both counted from the end of the
-          table. For example, slice(-2, -1) returns the second to last row of the table.
+            If the start is negative and the stop is negative, they are both counted from the end of the
+            table. For example, slice(-2, -1) returns the second to last row of the table.
```

Comment:
These spacing changes look potentially wrong.  I would expect spaces to be removed to get the left indent equal to the triple quotes.